.github/workflows/main.yml

```diff
@@ -79,8 +82,9 @@ jobs:
         with:
           ocaml-version: ${{ matrix.ocaml-version }}
           dune-cache: true
-          lint-fmt: true
-          opam-disable-sandboxing: true
+          mode: lint-fmt
```

Comment:
What's the effect of running `link-fmt` mode? (I see that we also have to run the format check ourselves explicitly, so just wondering what the `mode` does.)

Reply:
The mode had the functionality to find the vendored ocamlformat or to retrieve the required version from the ocamlformat config file to install ocamlformat.